Patch-Source: https://github.com/ryanoasis/nerd-fonts/pull/830
--
From 4f8711306428bf6371bcab963395d417bd89eec0 Mon Sep 17 00:00:00 2001
From: Fini Jastrow <ulf.fini.jastrow@desy.de>
Date: Sat, 14 May 2022 19:36:58 +0200
Subject: [PATCH] Fix Noto too wide

[why]
The 'monospace' width is determined by examining all the 'normal' glyphs
and taking the widest one.

'Normal' means 0x00-0x17f: the Latin Extended-A range.

Unfortunately Noto has the '1/2', '1/4', '3/4' that are all wider then the
normal (i.e. letter) glyphs.

[how]
Exclude a small sub-range from the 'find the widest glyph' that contain
no glyphs one would call 'letter'.

Fixes: #610

Signed-off-by: Fini Jastrow <ulf.fini.jastrow@desy.de>
---
 font-patcher | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/font-patcher b/font-patcher
index 7d063ab24..8e6470838 100755
--- a/font-patcher
+++ b/font-patcher
@@ -654,6 +654,8 @@ class font_patcher:
         #
         # 0x00-0x17f is the Latin Extended-A range
         for glyph in range(0x00, 0x17f):
+            if glyph in range(0x7F, 0xBF):
+                continue # ignore special characters like '1/4' etc
             try:
                 (_, _, xmax, _) = self.sourceFont[glyph].boundingBox()
             except TypeError:
